### PR TITLE
Debug

### PR DESCRIPTION
**Pull request by bapt**
many i_info should be i_debug to avoid doveadm to be too verbose in production

### CODE REVIEW

File: src/fts-backend-xapian.cpp [hunk 1]
```diff
@@ -482,7 +482,7 @@ static int fts_backend_xapian_lookup(struct fts_backend *_backend, struct mailbo
 	/* Performance calc */
         gettimeofday(&tp, NULL);
         dt = tp.tv_sec * 1000 + tp.tv_usec / 1000 - dt;
-	i_info("FTS Xapian: %ld results in %ld ms",n,dt);	
+	i_debug("FTS Xapian: %ld results in %ld ms",n,dt);
```
Comment (grosjo) [inline src/fts-backend-xapian.cpp:485]: Not sure this is adequate, becuase if one need to put dovecot in debug mode, then all the debug messages comes, and fts debug gets lost. ALso performance data is quite usefull not in debug mode.

File: src/fts-backend-xapian-functions.cpp [hunk 2]
```diff
@@ -452,7 +449,7 @@ static void fts_backend_xapian_oldbox(struct xapian_fts_backend *backend)
 		}
         	/* End Performance calculator*/
 	
-                i_info("FTS Xapian: Done indexing '%s' (%ld msgs in %ld ms, rate: %.1f)",backend->oldbox,backend->perf_nb,dt,r);
+                i_debug("FTS Xapian: Done indexing '%s' (%ld msgs in %ld ms, rate: %.1f)",backend->oldbox,backend->perf_nb,dt,r);
```
Comment (grosjo) [inline src/fts-backend-xapian-functions.cpp:452]: Not sure this is adequate, becuase if one need to put dovecot in debug mode, then all the debug messages comes, and fts debug gets lost. ALso performance data is quite usefull not in debug mode.

**Comment by bapt** [comment]
I do understand that, on the other hand, it makes the run of doveadm very verbose, making crontab and other commands that uses doveadm more verbose than expected with no way to turn it off. If you have ideas, I can implement them.

**Comment by grosjo** [comment]
@bapt to resolve teh matter, I would rather add a parameter in the config file (say "verbose true/false") which enables the debug info or not, so the administrator may decide if log info is neded or not for its own case

**Comment by bapt** [comment]
ok I will try to work on something like that when I do find the time.

**Comment by grosjo** [comment]
@bapt see recent changes (parameter "verbose")

**Comment by grosjo** [comment]
@bapt Let's continue on PR #27

**Pull request closed without merging**